app/store/menuItemState/menuItem.types.ts

```
@@ -1,24 +1,24 @@
-    isGlutenFree: boolean;  
-    isSpicy: boolean;
-    sortOrder: number;
-    createdAt: string;
-    updatedAt: string;
-    category: {
-      id: string;
-      name: string;
-    };
-    modifiers: any[];
+  };
+  modifiers: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)